tests/fixtures/decam.py

```diff
@@ -209,7 +214,8 @@ def decam_raw_image( decam_exposure ):
 def decam_small_image(decam_raw_image):
     image = decam_raw_image
     image.data = image.data[256:256+512, 256:256+512].copy()  # make it C-contiguous
-    return image
+
+    yield image
```

Comment:
Is there any benefit to using `yield` rather than `return` when there's nothing in the function after the `yield`?  (I'd think if anything there's a benefit in using `return`, as it keeps it simpler; you don't have issues of calling it inside other things.)

Reply:
Some of the fixtures are nested, and if you have a return instead of a yield, it messes up the cleanup of the containing fixture.